android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/authorization/DbContentProvider.java

```diff
@@ -1,56 +1,60 @@
+/*
+ * Copyright (c) 2022 Solana Mobile Inc.
+ */
+
 package com.solana.mobilewalletadapter.walletlib.authorization;
 
 import android.content.ContentValues;
 import android.database.Cursor;
 import android.database.sqlite.SQLiteDatabase;
 
-public abstract class DbContentProvider<T> {
-    public SQLiteDatabase mDb;
+/*package*/ abstract class DbContentProvider<T> {
+    private final SQLiteDatabase mDb;
 
-    public DbContentProvider(SQLiteDatabase db) {
+    /*package*/ DbContentProvider(SQLiteDatabase db) {
         this.mDb = db;
     }
 
-    protected abstract T cursorToEntity(Cursor cursor);
+    /*package*/ abstract T cursorToEntity(Cursor cursor);
```

Comment:
This method should probably stay `protected` - I'd not want anything other than a subclass to be able to invoke it.